procedures/katello/release.md.erb

```diff
@@ -26,12 +26,13 @@
   * [ ] Set version to `<%= version %>`
   * [ ] Set prerelease to `<%= extra %>`
 <% unless is_rc %>  * [ ] Reset release to `1`
-<% end %>  * [ ] In each package dir, remove the old tarball, run `spectool -g *.spec` and `git annex add *.tar.gz` or `git annex add *.gem`
+<% end %>  * [ ] Fetch new source by running `obal source rubygem-katello`, then `git rm` the old source.
```

Comment:
Have you looked at https://github.com/ekohl/theforeman-rel-eng/blob/master/bump_rpm_packaging? it does a lot of the same steps and intends to achieve the same thing.

Reply:
No, it could be used with some de-foreman-ification :)

Reply:
Yes, probably mostly around the source handling. I actually wrote a cleaner version in https://github.com/theforeman/foreman-packaging/blob/189fe820b1434174349d6bd6a6febecd541ab0b8/bump_rpm.sh#L53

Reply:
https://github.com/ekohl/theforeman-rel-eng/pull/4

Reply:
Can a good chunk of this subsection be replaced by using "obal update katello katello-repos rubygem-katello -e version=blah" ???

Reply:
I don't think obal can handle changing the `prerelease` macro. It might work for GA releases.

Reply:
I wonder then, if we could add that support to obal

Reply:
In https://github.com/ekohl/theforeman-rel-eng/pull/4#discussion_r337358631 we concluded: yes, but let's not block improvements on features obal doesn't have yet.

Reply:
created https://github.com/theforeman/obal/issues/210

Reply:
fixed in https://github.com/theforeman/obal/pull/213